### PR TITLE
Catch and log errors parsing CLA.json

### PR DESCRIPTION
**Pull request by ggetz**
The concierge should still post, and the error will be logged to stdout in the event the CLA request fails or does not parse.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@ggetz, thanks for the pull request! Maintainers, we have a signed CLA from @ggetz, so you can review this at any time.

I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium-concierge/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

:sparkles: :sparkles: :sparkles:__I'm the Cesium Concierge and I'm here to test things.__ :sparkles: :sparkles: :sparkles:

**Comment by mramato** [comment]
If I'm reading this right, yesterday's error with this change would end up not asking for the CLA, is that correct?  We definitely don't want that.  Perhaps we can make the PR comment indicate that concierge had a problem reading the CLA?

**Comment by ggetz** [comment]
> If I'm reading this right, yesterday's error with this change would end up not asking for the CLA, is that correct? We definitely don't want that. Perhaps we can make the PR comment indicate that concierge had a problem reading the CLA? 

Correct. I'll make the  comment indicate it instead.

File: lib/templates/pullRequestOpened.hbs [hunk 1]
```diff
@@ -1,11 +1,22 @@
 {{#if claEnabled}}
+{{#if errorCla}}
+@{{ userName }}, thanks for the pull request!
+
+If this is your first contribution, can you please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) so that we can review and merge this pull request?
```
Comment (mramato) [inline lib/templates/pullRequestOpened.hbs]: Sam comment as in https://github.com/AnalyticalGraphicsInc/cesium/pull/6359

File: lib/commentOnOpenedPullRequest.js [hunk 2]
```diff
@@ -33,13 +33,17 @@ function commentOnOpenedPullRequest(body, repositorySettings) {
...
+    var errorCla;
     return repositorySettings.fetchSettings()
         .then(function () {
             return commentOnOpenedPullRequest._askForCla(userName, repositorySettings);
         })
         .then(function (result) {
             askForCla = result;
         })
+        .catch(function(error) {
+            errorCla = error.toString();
```
Comment (mramato) [inline lib/commentOnOpenedPullRequest.js]: It looks like we treat this as a boolean, so shouldn't this just be `errorCla = true`?

Comment (ggetz) [reply lib/commentOnOpenedPullRequest.js]: > You can use the if helper to conditionally render a block. If its argument returns false, undefined, null, "", 0, or [], Handlebars will not render the block.

I do print the error, so I'm using the string if it's defined.

Comment (mramato) [reply lib/commentOnOpenedPullRequest.js]: Fair enough.

**Comment by ggetz** [comment]
Deployed.

**Pull request merged**